sounddevice.py

```diff
@@ -587,6 +587,38 @@ def get_status():
         raise RuntimeError('play()/rec()/playrec() was not called yet')
 
 
+def get_stream():
+    """Get a reference to the current Stream.
```

Comment:
I was referring to the fact that a `Stream` is a concrete class, but this is inconsistent with existing docstrings. Fixed.